packages/api/src/derives/nft/types.ts

```diff
@@ -21,3 +21,8 @@ export interface DeriveTokenInfo {
   /* Token owner address. It is undefined if the token is burnt */
   owner: string | undefined;
 }
+
+export interface CollectionMap {
```

Comment:
```suggestion
export interface CollectionInfo {
```
should include Id also